### PR TITLE
Fix #13; Fix #14; Reporting of WERs by index.htm

### LINKED ISSUE #13: Instrument DeepSpeech to graph training set size vs validation set WER

### LINKED ISSUE #14: Instrument DeepSpeech to graph hyperparameters vs validation set WER

### PR DESCRIPTION
**Pull request by tilmankamp**
@kdavis-mozilla : First version of the reporting page.

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
Pretty! LGTM
